src/vs/workbench/browser/actions/developerActions.ts

```diff
@@ -276,6 +285,21 @@ class ToggleScreencastModeAction extends Action2 {
 				}
 
 				if (!configurationService.getValue('screencastMode.onlyKeyboardShortcuts') || !titleLabel || shortcut?.commandId && (format === 'keys' || format === 'commandAndKeys' || format === 'commandWithGroupAndKeys')) {
+					// Fix label for arrow keys
+					if (keyLabel) {
+						if (keyLabel.includes('UpArrow')) {
```

Comment:
Are these `if` statements really needed?

Reply:
I think so. I don't know how showing arrow key names can be useful:

![2023-02-22_12-38](https://user-images.githubusercontent.com/100484/220674042-6f980bc5-ceff-43dc-9242-c9eaf5b8b15f.png)

Feel free to make any changes you think are best.